pg_search/migrations/0001_enable.py

```diff
@@ -0,0 +1,26 @@
+from django.conf import settings
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0001_initial"),
+    ]
+
+    database_setting = settings.DATABASES["default"]
```

Comment:
This looks unused, but I assume the plan is to fill in the `schema_name` field below?

Reply:
was just a mistake, cargo-culting from `pgroonga/migrations/0001_enable.py`. Fixed now